kafkakewl-deploy/src/main/scala/com/mwam/kafkakewl/deploy/endpoints/Endpoints.scala

```diff
@@ -19,11 +19,25 @@ class Endpoints(
   private val metricsEndpoint: PublicEndpoint[Unit, Unit, String, Any] =
     endpoint.in("metrics").get.out(stringBody)
 
+  // Health check endpoints. As of now, just return 200 no preparation is done
+  // after the HTTP server starts.
+  private val livenessEndpoint: PublicEndpoint[Unit, Nothing, Unit, Any] =
+    infallibleEndpoint.in("health").in("live").get
+  private val readinessEndpoint: PublicEndpoint[Unit, Nothing, Unit, Any] =
```

Comment:
Do we need 3 endpoints if all of them do exactly same thing for now? 
(I don't have a strong preference) but I think it would make more sense to add more endpoints if and when we need them (we can start with just one)

Reply:
one advantage of 3 endpoints is that external users can hook up their startup/readiness/liveness probes to the endpoins and if we want to have different implementation in the future, they just upgrade, no need to change their helmrelease/whatever deployment.